src/osd/PrimaryLogPG.cc

```diff
@@ -15,51 +15,41 @@
  *
  */
 
-#include <errno.h>
-
-#include <charconv>
-#include <sstream>
-#include <utility>
-
-#include <boost/intrusive_ptr.hpp>
-#include <boost/tuple/tuple.hpp>
```

Comment:
any good reasons to drop them?

Reply:
Already indirectly included by the other .h files

Reply:
i don't think "indirectly include" is the reason to remove them. IMHO, a .cc file should include the used header files, instead of relying included header files to do this.

Reply:
I don't think this is how we usually do things (apart from being wasteful in compilation time). There are tons
of include files mentioned in '.h' files that we do not repeat in the '.c'.  PG.h, for example, includes many files that we are no repeating in PG.cc. Nor in PrimaryLogPG.cc, event before my change.